demo/src/main.ts

```diff
@@ -88,7 +91,8 @@ const settingsModel = monaco.editor.createModel(
   "editor.semanticHighlighting.enabled": true,
   "editor.bracketPairColorization.enabled": false,
   "editor.fontSize": 12,
-  "audioCues.lineHasError": "on"
+  "audioCues.lineHasError": "on",
+  "audioCues.onDebugBreak": "on"
```

Comment:
Should setting break points work? I know this about audio config, but it implies break points should work. It does not here.

Reply:
What do you mean? You should be able to add breakpoint in the first editor, maybe I broke it?